src/main/java/io/cdap/e2e/utils/StorageClient.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.utils;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageException;
+import com.google.cloud.storage.StorageOptions;
+
+import java.io.IOException;
+
+/**
+ * Represents Google Cloud Storage client.
+ */
+public class StorageClient {
```

Comment:
We already have a `GCPClient` class as well! Either rename it to `BigQueryClient` or combine `GCStorageClient` and `BigQueryClient` in `GCPClient`

Reply:
`GcpClient` renamed to `BigQueryClient`